docs/source/conf.py

```diff
@@ -329,3 +322,24 @@
 # If false, no module index is generated.
 #
 # latex_domain_indices = True
+
+# Global variables for use in all pages
+#
+contact_email = 'dontknowyet@cisecurity.org'
+Maturity1 = 'Level 1'
```

Comment:
I'd like to stick with python variable naming conventions, if we can (i.e., lower, snake case).